base/applications/mmc/console.c

```diff
@@ -333,6 +345,12 @@ FrameOnCommand(HWND hwnd,
         case IDM_FILE_EXIT:
             PostMessage(hwnd, WM_CLOSE, 0, 0);
             break;
+			
+		case IDM_HELP_ABOUT:		
+			LoadAndFormatString(hAppInstance, IDS_APPTITLE, &lpTitle);
```

Comment:
`AllocAndLoadString()` should be enough.

Reply:
Done but now it fails at compilation.... any clue ?